nextflow_schema.json

```
@@ -87,7 +87,8 @@
                     "type": "string",
                     "default": "unicycler",
                     "fa_icon": "fas fa-puzzle-piece",
-                    "description": "The assembler to use for assembly. Available options are `Unicycler`, `Canu`, `Miniasm`. The latter two are only available for long-read data, whereas Unicycler can be used for short or hybrid assembly projects."
+                    "description": "The assembler to use for assembly. Available options are `Unicycler`, `Canu`, `Miniasm`, or `Dragonflye`. The latter trhee are only available for long-read data, whereas Unicycler can be used for short or hybrid assembly projects.",
```

```suggestion
                    "description": "The assembler to use for assembly. Available options are `Unicycler`, `Canu`, `Miniasm`, or `Dragonflye`. The latter three are only available for long-read data, whereas Unicycler can be used for short or hybrid assembly projects.",
```
If you want to make it easier for future changes, I would recommend to not list the available options in the description, so the don't get out of sync with the options listed in the `enum`